### DISCUSSION TITLE
Matomo support, just as Google Analytics support

### DISCUSSION THREAD

**Discussion by MaxValue** [outline/outline#5087] [Ideas]
It would be really cool if Outline supported Matomo (Piwik), just as it supports Google Analytics. Especially since Matomo is open-source, just as is Outline.

- It would an entry under "Integrations" called "Matomo"
- This entry would need the following config keys
    - **Matomo Instance URL**: The URL of the desired Matomo instance
    - **Site ID**: The ID of the site inside the Matomo instance
- It would probably be best to use Matomo’s API to add new tracking data, since the Google Analytics integration also adds additional details, which would probably not be possible via the simple embeddable tracking script.

**Comment by tommoor** [comment]
Definitely open to more analytics integrations, there are quite a few Google Analytics alternatives these days and It would be good to give preference to those that can be self hosted.

Ideally we need to move the current Google Analytics integration to the new `plugins` format first and that can provide a base/template for others.
